client/src/Main.elm

```diff
@@ -1189,16 +1292,16 @@ insertTable model =
                     , { header = none
                       , width = fillPortion 5
                       , view =
-                            \insert ->
-                                Input.button [ Font.color color.blue, Font.bold, Font.underline ] { onPress = Just (AppendInsert insert), label = Element.text insert.name }
+                            \level0 ->
+                                Input.button [ Font.color color.blue, Font.bold, Font.underline ] { onPress = Just (AppendInsert level0), label = Element.text level0.name }
```

Comment:
This line is too long. You can add a newline somewhere and `elm-format` will help.

Reply:
Done!